tests/backup/backup_test.go

```diff
@@ -143,6 +143,110 @@ var _ = Describe("{UserGroupManagement}", func() {
 	})
 })
 
+// This is to create multiple users and groups
+var _ = Describe("{CreateMultipleUserAndGroup}", func() {
+	numberOfUsers := 20
+	numberOfGroups := 10
+	users := make([]string, 0)
+	groups := make([]string, 0)
+	groupNotCreated := make([]string, 0)
+	userNotCreated := make([]string, 0)
+
+	JustBeforeEach(func() {
+		wantAllAfterSuiteActions = false
+		StartTorpedoTest("CreateMultipleUserAndGroup", "Creation of multiple users and groups", nil, 0)
```

Comment:
Provide the testrail ID of the test case here instead of `0`

Typo - It should be MultipleUsersAndGroups

Reply:
Added the testrail Id